CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# Contributing to this repository
+
+## Contributions in English
+
+### Using Bold
+- to note step numbers. Example: __First__, __Second__, __Third__, etc
+- to designate important Creator Components, Objets, etc. Example: __Scene__, __Sprite Component__
+- to designate important Creator UI elements. Example: __Hierarchy Panel__, __Assets Panel__
+
+### Using Italics
+- list items. Example: *apple*, *orange*, *banana*, and *pear*
```

Comment:
Why is that ? I don't see any advantage

Reply:
list items, I think it's okay to not add styles. For example:

![image](https://user-images.githubusercontent.com/19702899/83228710-5bd61580-a1b9-11ea-9db2-453a79676633.png)

Or maybe a bold here would be fine? I think it's better not to have too many types of styles?

Reply:
I agree, just leave them in the normal form is ok, at least it shouldn't be a rule, if the writer think it's better to use bold or italic to distinguish them, he can, but he is not forced to.